pandas/tests/extension/json/test_json.py

```diff
@@ -356,9 +356,10 @@ def test_groupby_extension_no_sort(self):
         """
         super().test_groupby_extension_no_sort()
 
-    @pytest.mark.xfail(reason="GH#39098: Converts agg result to object")
-    def test_groupby_agg_extension(self, data_for_grouping):
-        super().test_groupby_agg_extension(data_for_grouping)
+    # GH#39098 is now fixed, so we don't need to XFAIL, nor subclass this test
+    # @pytest.mark.xpass(reason="GH#39098: Converts agg result to object")
+    # def test_groupby_agg_extension(self, data_for_grouping):
+    #     super().test_groupby_agg_extension(data_for_grouping)
```

Comment:
I think all we need to do is remove the comment and the decorator and let the testsuite do its thing (passing if the test passes, which it should, and failing if the test fails, which it now should not).  I can push that through, but will await other comments first, to save an hour's worth of CI/CD time in case there are other comments.